pantheon-bundle/frontend/src/app/assemblyDisplay.tsx

```diff
@@ -143,6 +144,7 @@ class AssemblyDisplay extends Component<any, any, any> {
                         <Card>
                             <Versions
                                 modulePath={this.state.modulePath}
+                                type="assembly"
```

Comment:
please create an entry for assembly in pantheon-bundle/frontend/src/app/Constants.tsx